src/app/_components/chat/chatroom/ChatItem.tsx

```diff
@@ -3,23 +3,24 @@ import Message from './Message'
 import { extractTimeFromDate } from '@/utils/date'
 
 interface ChatItemProps {
+  user?: SimpleUser
   newSender?: boolean
   type: 'YOU' | 'ME'
-  chat: ChatMessageFromServer | ChatMessageFromClient
+  chat: ChatMessageFromServer
 }
 
-const ChatItem = ({ newSender, type, chat }: ChatItemProps) => {
+const ChatItem = ({ newSender, type, chat, user }: ChatItemProps) => {
```

Comment:
```suggestion
const ChatItem = ({ newSender, type, chat: { content, createdAt, isRead  }, user }
```

`chat`은 아래에서 안 쓰이므로 이렇게 선언해도 좋아요.

Reply:
넵 수정했습니다!